examples/policies/multi-asset-policy/multi-asset-policy.pact

```diff
@@ -144,7 +154,10 @@
 
       (map 
         (lambda (asset-id:integer)
-          (emit-event (ASSET_REJECTED token-id asset-id (at asset-id assets)))
+          (with-capability (ASSET_REJECTED token-id asset-id (at asset-id assets) owner)
```

Comment:
Here, the rejector has to calculate all the asset-id's and assets when signing the capability to reject ALL proposed assets, so wouldn't it make sense to create something like `ASSET_REJECTED_ALL` that takes in `token-id` and `owner`?